hail/python/hail/experimental/ldscsim.py

```diff
@@ -285,49 +317,62 @@ def multitrait_ss(mt, h2, pi, rg=0, seed=None):
         Genetic correlation between traits.
     seed : :obj:`int`, optional
         Seed for random number generator. If `seed` is ``None``, `seed` is set randomly.
-    
+
     Warning
     -------
     May give inaccurate results if chosen parameters make the covariance matrix 
     not positive semi-definite. Covariance matrix is likely to not be positive
-    semi-definite when :math:`p_{TT}` is small and rg is large.
-    
+    semi-definite when :mat:`p_{TT}` is small and rg is large.
```

Comment:
typo: this is causing a docs build error

Reply:
argh, sorry about that. It should now be fixed